.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, in
```

Comment:
I *think* this should be "at URL". @mwiencek?

Reply:
Same as above (and above above), this is from [Picard’s PR template](https://github.com/metabrainz/picard/blob/master/.github/PULL_REQUEST_TEMPLATE.md), thus I requested a review from @zas

Reply:
I'd probably have used "at" if I wrote it, though I'm not sure "in" is technically incorrect or anything.

Reply:
Changed by https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/e3588b1d5acbac6fec0b6a5bdc4c41fc25635642